### PR TITLE
Fix HTTP headers for USSD app

### PR DESCRIPTION
**Pull request by rudigiesler**
In a previous PR, we added User Agent headers to http requests, so that we can track which apps are making requests. This was added incorrectly, the PR fixes that.

### CODE REVIEW

File: src/ussd_nurse.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ go.app = function() {
         var engage;
 
         self.init = function() {
-            var config = {headers: {'User-Agent': 'Jsbox/NDoH-Nurse'}};
+            var config = {headers: [{'User-Agent': 'Jsbox/NDoH-Nurse'}]};
```
Comment (erikh360) [inline src/ussd_nurse.js]: Is the list in the right place here?

**Review by erikh360 — COMMENTED** [review]
Just the one question.

File: src/ussd_nurse.js (hunk 1 above)
Comment (rudigiesler) [reply src/ussd_nurse.js]: Ah, it's not, will fix

Comment (rudigiesler) [reply src/ussd_nurse.js]: Although we should probably remove this app, since we've moved over to RapidPro

**Comment by rudigiesler** [comment]
@erikh360 Fixed. Ready for another review

**Review by erikh360 — APPROVED** [review]
Cool!, we should create a maintenance ticket to remove the NC things from seed.

**Pull request merged**